.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,55 @@
-<!-- Please feel free to remove whatever sections/lines in this aren’t relevant.
-
-Use the title line as the title of your pull request, then delete these lines.
-
-## Title line template: [Title]: Brief description
-
-Website: For pull requests that impact designsystem.digital.gov’s look, feel, or functionality, please open a pull request on the uswds-site repo (https://github.com/uswds/uswds-site).
-
+<!---
+Welcome! Thank you for contributing to the U.S. Web Design System.
+Your contributions are vital to our success and we are glad you're here.
+
+Please keep in mind:
+- This pull request (PR) template exists to expedite integration. The USWDS Core team reviews and approves every PR before merging it into the public code base, so the better we understand the problem and proposed solution, the sooner we can merge this change. The point here is: clear explanations matter!
+- You can see examples of successful PRs that meet our criteria by ... [TK?]
+- You can erase any part of this template that doesn't apply to your pull request (including these instructions!).
+
+If you get stuck, you can find more information about contributing in contributing.md or you can reach out to us directly at uswds@gsa.gov.
+ -->
+
+<!---
+Step 1: Add a title
+Title this PR with the following format:
+USWDS - [Package]: [Brief statement describing what this pull request solves]
+eg: "USWDS - Button: Increase font size"
+ -->
+
+## Related issue
+<!-- Every pull request should resolve an open issue.
+If no open issue exists, you can open one here: https://github.com/uswds/uswds/issues/new/choose) -->
+Closes [Issue #]
+
+## Preview link
+<!-- If available, provide a link to a demo of of the solution in action.-->
+
+## Problem statement
+<!-- Summarize the problem this PR solves in a clear and concise statement.
+A successful problem statement conveys:
+1. The desired state
+2. The actual state
+3. Consequences of current state (who does this affect and to what degree?)-->
+
+## Solution
+<!-- Provide a summary of the solution this PR offers.
+It can be helpful if we understand:
+1. What the solution is,
+2. Why this approach was chosen,
+3. How you implemented the change, and
+4. Possible limitations of this approach and alternate solution paths.
 -->
```

Comment:
```suggestion
<!-- 
Provide a summary of the solution this PR offers.

It can be helpful if we understand:
1. What the solution is,
2. Why this approach was chosen,
3. How you implemented the change, and
4. Possible limitations of this approach and alternate solution paths.
-->
```